packages/codesniffer/tests/php/bootstrap.php

```diff
@@ -5,6 +5,11 @@
  * @package automattic/jetpack-codesniffer
  */
 
+if ( version_compare( PHP_VERSION, '7.2.0', '<' ) ) {
+	echo 'PHP version is too old to run tests. 7.2 is required, but ' . PHP_VERSION . " is installed.\n";
+	exit( 1 );
```

Comment:
`exit(1)` here makes the 5.6 and 7.0 tests fail, because it exited with a non-zero exit code.

We could do `exit(0)`. Or we could test `getenv( 'CI' ) === 'true'` (or any of the other environment variables listed [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables), or set one of our own) to determine which exit code to use to be a little more semantically correct.

Reply:
Actually, that's not why it was failing. It fails even before that when composer does `@composer install`.

Reply:
<details>
<summary>So the best I can come up with is to do something like this diff</summary>

```diff
diff --git a/.github/workflows/phpunit.yml b/.github/workflows/phpunit.yml
index 260a348ef..66e3028ac 100644
--- a/.github/workflows/phpunit.yml
+++ b/.github/workflows/phpunit.yml
@@ -84,7 +84,7 @@ jobs:
             cd "$PACKAGE/../.."
             NAME=$(basename $(pwd))
             echo "::group::Package $NAME"
-            composer phpunit
+            composer phpunit || [[ $? -eq 77 ]]
             echo "::endgroup::"
             cd ../..
           done
diff --git a/packages/codesniffer/composer.json b/packages/codesniffer/composer.json
index 228edcde2..a9d4d09f6 100644
--- a/packages/codesniffer/composer.json
+++ b/packages/codesniffer/composer.json
@@ -20,6 +20,7 @@
        },
        "scripts": {
                "phpunit": [
+                       "@php -r 'exit( version_compare( PHP_VERSION, \"7.2\", \">=\" ) ? 0 : 77 );'",
                        "@composer install",
                        "./vendor/phpunit/phpunit/phpunit --colors=always"
                ]
```
The 77 is arbitrary, we just don't want to collide with an exit code something might legitimately use.
</details>

Or else just keep the magic script somewhere and just rename it to not have "travis" in the name.

Reply:
I see 101f9e3 addressed this.